src/fs/filesystem.ts

```diff
@@ -172,94 +208,83 @@ export class FileSystem {
   }
 
 
+  async updateRootLink(branch: Branch, update: AddResult): Promise<void> {
+    this.root.updateLink(link.make(branch, update.cid, false, update.size))
+  }
+
 
   // POSIX INTERFACE
   // ---------------
 
-  async add(path: string, content: FileContent): Promise<CID> {
+  async mkdir(path: string): Promise<CID> {
```

Comment:
Actually about line 216 — anything that you can make a logic section about (like “POSIX interface”) can be a TS `interface`